release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -572,6 +572,14 @@ If you are receiving pod security violations, see the following resources:
 
 The {cert-manager-operator} 1.11 is now generally available in {product-title} 4.14 as well as {product-title} 4.13 and {product-title} 4.12.
 
+[discrete]
+[id="ocp-4-14-triggered-updates-for-etcd-backups"]
+=== Updates to z releases no longer require an etcd backup
+
+An earlier link:https://github.com/openshift/cluster-etcd-operator/pull/835[bug fix] introduced a backup trigger on every z release. However, the Cluster Version Operator (CVO) logic was not updated along with that bug fix, and as a result, the update process stopped until an etcd snapshot was taken. This behavior affected versions 4.10 and later. The backup trigger is no longer enabled, so you can now update to a z release without taking an etcd snapshot.
+
```

Comment:
> The backup trigger is no longer enabled, so you can now update to a z release without taking an etcd snapshot.

you can leave this out, the snapshot was taken by the operator automatically and it was opaque to the user.